apis/core.oam.dev/v1alpha2/application_types.go

```diff
@@ -88,7 +88,7 @@ type ApplicationSpec struct {
 
 // Application is the Schema for the applications API
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:categories={oam},shortName=apps
+// +kubebuilder:resource:categories={oam},shortName=app;apps
```

Comment:
apps is enought, `app` not  necessary and could cause conflict with other CRD easily

Reply:
`apps` will conflict with `argoproj.io` as follows. I added `app`because short name is usually a singular word

![image](https://user-images.githubusercontent.com/4902714/114017684-e4741680-989e-11eb-9673-b12348f7d562.png)

Reply:
If so, I agree `app` , we could remove `apps`

Reply:
Done